Documentation/BuildInstructions.md

```diff
@@ -11,11 +11,11 @@ sudo apt install build-essential cmake curl libmpfr-dev libmpc-dev libgmp-dev e2
 ```
 Optional: `fuse2fs` for [building images without root](https://github.com/SerenityOS/serenity/pull/11224).
 
-#### GCC 12 or Clang 17
+#### GCC 13 or Clang 17
 
-A host compiler that supports C++23 features is required for building host tools, the newer the better. Tested versions include gcc-12 and clang-17.
+A host compiler that supports C++23 features is required for building host tools, the newer the better. Tested versions include gcc-13 and clang-17.
 
-On Ubuntu gcc-12 is available in the repositories of 22.04 (Jammy) and later.
+On Ubuntu gcc-13 is available in the repositories of 22.04 (Jammy) and later.
```

Comment:
gcc 13 requires the toolchain-test PPA on 22.04. it (should be?) available on 24.04 though

Reply:
Fixed